tools/dump_tool/HowToUseDumpTool.md

```diff
@@ -272,6 +272,35 @@ python dumpParser.py -r $TIZENRT_BASEDIR/build/output/bin/ramdump_0x02020000_0x0
 ```
 2. See the Output
 
+#### To display Debug Symbols
```

Comment:
Could you add description about the option like `-e'?
There are many options, `-e`, `-t` and `-g`.
It is better to use if there is an explanation.

Reply:
These usages show up in case of a wrong argument. It is in the ramdumpParser script itself